modin/core/dataframe/base/partitioning/axis_partition.py

```diff
@@ -114,9 +117,9 @@ def force_materialization(self, get_ip=False):
         materialized = self.apply(
             lambda x: x, num_splits=1, maintain_partitioning=False
         )
-        return type(self)(materialized, get_ip=get_ip)
+        return type(self)(materialized, get_ip=get_ip)  # type: ignore[call-arg]
```

Comment:
Why do we need the ignore?

Reply:
Because we have not yet defined an `__init__` function in class `BaseDataframeAxisPartition` with the args `materialized` and `get_ip`, mypy is outputting the following messages:
```
modin/core/dataframe/base/partitioning/axis_partition.py:120:16: error: Too many arguments for "BaseDataframeAxisPartition"  [call-arg]
modin/core/dataframe/base/partitioning/axis_partition.py:120:16: error: Unexpected keyword argument "get_ip" for "BaseDataframeAxisPartition"  [call-arg]
```

If we implement the following method, mypy doesn't emit any warning
```python
def __init__(self, *args: Iterable, **kwargs: bool) -> None:
        super().__init__()
```